guides/README.md

```diff
@@ -67,6 +67,19 @@ It's also possible to check links; the following command will check all links ex
 
 	make linkchecker
 
+## Building Locally Using a Container
+
```

Comment:
Can we cater to the lowest common denominator because some writers might not be as technical as you, and just add that ensure that you have Docker installed.

Reply:
I've added minimal usage instructions. Afterall, using a container should make things easier.

Reply:
Thanks. I get you completely. And am totally satisfied with the minimal instructions. I think that we are covered by this.